modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/constants/FDSSampleFDSNames.java

```diff
@@ -16,6 +16,9 @@ public class FDSSampleFDSNames {
 	public static final String CONTROLLED =
 		FDSSamplePortletKeys.FDS_SAMPLE + "-controlled";
 
+	public static final String CUSTOM_INTERNAL_VIEW =
+		FDSSamplePortletKeys.FDS_SAMPLE + "-custom-internal-view";
```

Comment:
maybe, same way we have "custom cell renderer" we can use "custom view renderer" but it sounds too close to "custom view" as well. But indeed, what is custom is not the cell or the view, it's the renderer, so a more awkward but correct name would be "view custom renderer" for this thing. Analogously "cell custom renderer".

That leaves space for "custom views" as a separate end user feature

Reply:
The renderer falls under `customRenderers.view`, variable being `carouselViewRenderer`. I think that is reasonable, I'm only thinking to replace `view` with `views`.

The awkwardness comes when you go past `*Renderer`, because renderer is only an internal mechanism. What is the feature itself, that the render is accomplishing? It is a new version of `FDSView`, or in frontend, a new version of a `view`. You could say... a "custom view". But that's already taken. So I came up with "custom internal view", differentiating it from "custom user view", that would be current "custom view" in UI.